INSTALL.md

```diff
@@ -69,6 +69,24 @@ at [localhost:1081](http://localhost:1081).
 - Keep an eye on console in case of compiling errors.
 - [Read more](https://github.com/Trustroots/trustroots/wiki/Development)
 
+## Enable collecting statistics
+
+* 1. [Install InfluxDB](https://docs.influxdata.com/influxdb/latest/introduction/installation/) 0.9+ and run it (type `influxd`) at default port (8086)
```

Comment:
I don't like using public dev services as they might go away, or have a different version, etc... adds more entropy into my dev environment.

Seems it could be an optional step, if you are working on that bit of the code it would be useful, otherwise, yes it adds complexity to the setup for not much gain.